src/utils/disable-download.ts

```diff
@@ -0,0 +1,9 @@
+import semver from 'semver';
+
+export function disableDownload(version: string) {
+  return (
+    process.platform === 'darwin' &&
+    process.arch === 'arm64' &&
+    semver.lt(version, '11.0.0')
+  );
```

Comment:
To improve the stability of subsequent unit tests, please change to:

```suggestion
  return (
    !process.env.JEST &&
    process.platform === 'darwin' &&
    process.arch === 'arm64' &&
    semver.lt(version, '11.0.0')
  );
```

See:
* https://github.com/electron/fiddle/pull/1022
* https://github.com/electron/fiddle/pull/992

Reply:
Enabling the above change would break the test that I had written for disableDownload. Hence, I have ```deleted my test for utils/disable-download.ts```. I have, however, ```added component tests to components versionSelect and ElectronSettings``` which use disableDownload function. The test that I had written for disableDownload is as follows and  would always fail (no matter what) for MacOS since ```!process.env.JEST``` will be returning false here.

![Screenshot from 2022-03-22 20-00-48](https://user-images.githubusercontent.com/76606666/159508521-7c1e5bcc-89ea-4858-8192-160b67cedda1.png)

Reply:
Sure. Will try to think of a solution as well.

Reply:
I am writing the code for this and I will submit it later

Reply:
Done.
See: https://github.com/electron/fiddle/pull/1019/commits/8b83a71c81afbef4ba92b27e378d93231d562154

> I have committed the code to the current PR. you need to synchronize locally (pull)

Reply:
Unit test pass: https://github.com/electron/fiddle/runs/5656821882?check_suite_focus=true#step:6:75 🎉

Reply:
Great!!